### PR TITLE
Updating to project notification

### PR DESCRIPTION
**Pull request by KitArmstrong**
Ticket: https://bcmines.atlassian.net/browse/EE-873
Changes:
- Updates post/put endpoints to handle the new project notification model

Relates to Admin PR https://github.com/bcgov/eagle-admin/pull/509

### CODE REVIEW

File: api/controllers/projectNotification.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+  if (args.swagger.params.publish && args.swagger.params.publish.value) {
+    projectNotification.read.push('public');
+  }
+
+  try {
+    const saveProjectNotification = await projectNotification.save();
+    Utils.recordAction('Post', 'ProjectNotification', args.swagger.params.auth_payload.preferred_username, saveProjectNotification._id);
+    defaultLog.info('Saved new project notification object:', saveProjectNotification);
+    return Actions.sendResponse(res, 200, saveProjectNotification);
```
Comment (dhlevi) [inline api/controllers/projectNotification.js]: Response code for a create should be a 201

File: api/swagger/swagger.yaml [hunk 2]
```diff
@@ -4649,23 +4647,23 @@ paths:
         "200":
```
Comment (dhlevi) [inline api/swagger/swagger.yaml]: Same as controller, expected response should be a 201

File: api/controllers/projectNotification.js (hunk 1 above)
Comment (KitArmstrong) [reply api/controllers/projectNotification.js]: Updated

File: api/swagger/swagger.yaml (hunk 2 above)
Comment (KitArmstrong) [reply api/swagger/swagger.yaml]: Updated

**Pull request merged**